modules/admin_manual/pages/configuration/general_topics/search.adoc

```diff
@@ -67,32 +67,77 @@ For the URL, both `HTTP` and `HTTPS` incuding the address and port can be used.
 
 The app provides several authentication methods. Select the one of your choice and check out the details for the respective authentication method below:
 
-image:apps/search_elastic/search_elastic_auth_items.png[Auth Items,width=450]
+image::apps/search_elastic/search_elastic_auth_items.png[Auth Items,width=450]
 
 ==== No Authentication
 
 When using `No Authentication`, just fill in the URL of the ES server.
 
-image:apps/search_elastic/search_elastic_no_auth.png[No Auth,width=350]
+image::apps/search_elastic/search_elastic_no_auth.png[No Auth,width=350]
 
 ==== User / Password Authentication
 
 When using `User / Password Authentication`, enter the credentials set up on the ES server. Note that the password will be stored encrypted in the ownCloud database.
 
-image:apps/search_elastic/search_elastic_u_p_auth.png[User / Password Auth,width=350]
+image::apps/search_elastic/search_elastic_u_p_auth.png[User / Password Auth,width=350]
 
 ==== API Key Authentication
 
 When using `API Key Authentication`, enter the API Key with which the ES server was set up.
 
-image:apps/search_elastic/search_elastic_api_key_auth.png[API Key Auth,width=350]
+image::apps/search_elastic/search_elastic_api_key_auth.png[API Key Auth,width=350]
 
 IMPORTANT: The API Key needs to be the _encoded_ one, *not* the _api_key_ string. For details see the {create-api-key-url}[Create API key API] at the ES documentation.
 
 === Search External Storage
 
 Define if external storage should be included in ES indexing by setting the checkmark accordingly with btn:[Scan external Storages]. Setting this checkmark not only enables search in external storages, but also search in federated shares. Note that this setting requires to rebuild the index.
 
+=== Connector Setup
+
+There are 2 different indexes. The `Legacy` is the old / current one while the `RelevanceV2` is the new one. The difference between the two is how index data is stored because both provide different capabilities and index in different ways. This results in different search capabilities. The Legacy doesn't have any difference with the current search while the RelevanceV2 has new capabilities.
```

Comment:
```suggestion
There are 2 different indexes. The `Legacy` is the old / current one while the `RelevanceV2` is the new one. The difference between the two is how index data is stored because both provide different capabilities and index in different ways. This results in different search capabilities. The Legacy doesn't differ from the current search while the RelevanceV2 has new capabilities.
```